frontend/package.json

```diff
@@ -28,9 +28,10 @@
   "dependencies": {
     "@next/third-parties": "^14.3.0-canary.59",
     "@opentelemetry/api": "^1.8.0",
-    "@trussworks/react-uswds": "^7.0.0",
-    "@uswds/uswds": "^3.6.0",
+    "@trussworks/react-uswds": "^9.0.0",
+    "@uswds/uswds": "^3.7.1",
     "clsx": "^2.1.1",
+    "focus-trap-react": "^10.2.3",
```

Comment:
a new peer dependency of Trussworks that we need to include manually with the latest version